webpack.config.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
   externals: [
     // Treat all sdk requires as importable commonjs modules.
     (context, request, callback) => {
-      if (/^sdk\//.test(request)) {
+      if (/^sdk\//.test(request) || /^resource\:\/\//.test(request)) {
```

Comment:
Probably not as readable, but you could condense this into a single RegExp, something like:

```js
if (/^(sdk\/|resource:\/\/)/.test(request)) {
```

Although it's only a webpack build, so possibly not worth the complexity.